### ISSUE TITLE
History is not showing latest commit

### ISSUE THREAD

**Issue by BruceEckel** [desktop/desktop#10046]
### Describe the bug

History is not showing latest commit. I have verified that the commit has appeared on the Github site.

### Version & OS

2.5.2

Windows 10

### Steps to reproduce the behavior

Make a commit and then look at the history

### Expected behavior

Shows the changes I just checked in

### Actual behavior

What I see are yesterday's commits, not the one I just did.

**Comment by tierninho** [comment]
@BruceEckel Thanks for letting us know. Can you share your logs -- see the Help menu in Desktop. I suspect another issue is preventing the fetching of the latest commits.

**Comment by BruceEckel** [comment]
Attached.

I just thought of another issue: I discovered yesterday that Microsoft
OneDrive was causing problems, and removed it (following their
instructions, so it really is a mess). I noticed that it removed a bunch of
files, which I was able to restore using Github Desktop.

But there might be other files it is unable to reach. Ideally it would not
just quietly fail, though.

*Bruce Eckel*
MindViewLLC.com
Blog: BruceEckel.com
EvolveWork.co
WinterTechForum.com <http://www.WinterTechForum.com>
OnJava8.com <http://www.OnJava8.com>
www.AtomicKotlin.com
Reinventing-Business.com <http://www.Reinventing-Business.com>
